app/src/main/java/pm/gnosis/heimdall/ui/settings/SettingsActivity.kt

```diff
@@ -36,6 +37,12 @@ class SettingsActivity : BaseActivity() {
         layout_settings_tokens.setOnClickListener {
             startActivity(TokenManagementActivity.createIntent(this))
         }
+
+        if (encryptionManager.canSetupFingerprint()) {
```

Comment:
shouldn't we completely hide the row it it is not possible to setup a fingerprint?